nifi-extension-bundles/nifi-jolt-bundle/nifi-jolt-processors/src/test/java/org/apache/nifi/processors/jolt/TestJoltTransformJSON.java

```diff
@@ -464,6 +464,71 @@ void testJoltSpecInvalidEL() throws IOException {
         runner.assertNotValid();
     }
 
+    @Test
```

Comment:
@exceptionfactory I see 4 new unit tests which are pretty much configured very similarly. Do you think these four tests should be folded into a JUnit5 `ParamaterizedTest` ?